src/test/java/org/mitre/synthea/export/JSONExporterTest.java

```diff
@@ -0,0 +1,34 @@
+package org.mitre.synthea.export;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mitre.synthea.TestHelper;
+import org.mitre.synthea.engine.Generator;
+import org.mitre.synthea.helpers.Config;
+import org.mitre.synthea.world.agents.Person;
+
+public class JSONExporterTest {
+  @Rule
+  public TemporaryFolder tempFolder = new TemporaryFolder();
+
```

Comment:
Could you also add a test for the "exporter.json.include_module_history" property? That seems like the kind of thing where someday a refactor could break that feature.